wagtail/contrib/search_promotions/models.py

```diff
@@ -9,15 +9,31 @@
 
 
 class Query(models.Model):
+    """
+    Query model for search promotions
+
+    Attributes
+
+       - query_string (forms.CharField): Query text to match.
+    """
+
```

Comment:
```suggestion
```

Please remove this docstring, it does not add any value to the context of this PR. These can be added later as a different PR if you feel it's adding to the code usefulness.